usaspending_api/financial_activities/models.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from usaspending_api.accounts.models import AppropriationAccountBalances
+from usaspending_api.accounts.models import AppropriationAccountBalances, TreasuryAppropriationAccount
```

Comment:
This PR makes AppropriationAccountBalances an unused import... we don't have anything automated in our toolchain for detecting these, actually.  I should make a ticket for looking into that, we've probably got a ton of them.